### DISCUSSION TITLE
What is IgGP?

### DISCUSSION THREAD

**Discussion by vincentwalter** [milaboratory/mixcr#1745] [Q&A] [answered]
When exporting isotypes from a cloneset I get the following isotypes:

- IgA
- IgA1
- IgA2
- IgD
- IgE
- IgG
- IgG1
- IgG2
- IgG3
- IgG4
- **IgGP**
- IgM

I am not aware of an isotype called IgGP and a quick google search does not produce meaningfull results.
Neither does the Janeway list such an isotype.
I searched the MiXCR docs but they do not elaborate on this topic.
Could you please explain what this means?

Thank you very much!

**Comment by mizraelson** [comment, marked as answer]
Hi, 
IgGP stand for the non-functional [IGHGP gene](https://www.genecards.org/cgi-bin/carddisp.pl?gene=IGHGP).

**Reply by vincentwalter** [reply]
Thank you for the quick and helpful reply. A quick follow up, though.

I got this using the `cellecta-human-rna-xcr-umi-drivermap-air` preset.
How can these sequences that originated from RNA and are labelled productive be based on a pseudogene?

**Reply by mizraelson** [reply]
It is hard to tell without knowing the details. To my knowledge pseudogenes can be expressed on the mRNA level in some cases.
